tools/metanovo/metanovo.xml

```diff
@@ -199,108 +202,108 @@ novor_mass_analyzer=Trap
-      <param name="annotation_high_resolution" argument="-annotation_high_resolution" label="If true the most accurate peak will be selected within the m/z tolerance." truevalue="1" falsevalue="0" type="boolean" checked="true"/>
+      <param argument="-annotation_level" label="The intensity threshold to consider for annotation" value="0.75" type="float" optional="true" help="Using percentiles, 0.75 means that the 25% most intense peaks will be annotated."/>
+      <param argument="-annotation_high_resolution" label="If true the most accurate peak will be selected within the m/z tolerance." truevalue="1" falsevalue="0" type="boolean" checked="true"/>
     </section>
     <section name="sequence_matching" expanded="False" title="Sequence Matching">
-      <param name="sequence_index_type" argument="-sequence_index_type" label="sequence_index_type (deprecated)" value="0" type="integer" optional="true"/>
-      <param name="sequence_matching_type" argument="-sequence_matching_type" label="The peptide to protein sequence matching type" type="select">
+      <param argument="-sequence_index_type" label="sequence_index_type (deprecated)" value="0" type="integer" optional="true"/>
+      <param argument="-sequence_matching_type" label="The peptide to protein sequence matching type" type="select">
           <option value="0">Character Sequence</option>
           <option value="1">Amino Acids</option>
           <option selected="true" value="2">Indistinguishable Amino Acids</option>
       </param>
-      <param name="sequence_matching_x" argument="-sequence_matching_x" label="The maximal share of Xs in a sequence, 0.25 means 25% of X's" value="0.25" type="float" optional="true"/>
+      <param argument="-sequence_matching_x" label="The maximal share of Xs in a sequence, 0.25 means 25% of X's" value="0.25" type="float" optional="true"/>
     </section>
     <section name="import_filters" expanded="False" title="Import Filters">
-      <param name="import_peptide_length_min" argument="-import_peptide_length_min" label="The minimal peptide length to consider when importing identification files" value="8" type="integer" optional="true"/>
-      <param name="import_peptide_length_max" argument="-import_peptide_length_max" label="The maximal peptide length to consider when importing identification files" value="30" type="integer" optional="true"/>
-      <param name="import_precursor_mz_ppm" argument="-import_precurosor_mz_ppm [sic]" label="Maximal precursor ion deviation unit" type="select">
+      <param argument="-import_peptide_length_min" label="The minimal peptide length to consider when importing identification files" value="8" type="integer" optional="true"/>
+      <param argument="-import_peptide_length_max" label="The maximal peptide length to consider when importing identification files" value="30" type="integer" optional="true"/>
+      <param argument="-import_precurosor_mz_ppm" label="Maximal precursor ion deviation unit" type="select">
```

Comment:
```suggestion
      <param argument="-import_precursor_mz_ppm" label="Maximal precursor ion deviation unit" type="select">
```

Looks like I had a typo on the original `argument` attribute. This should fix the current test failure.

Reply:
Thanks for adding this and making updates for the linter!